flang/lib/Optimizer/CodeGen/TargetRewrite.cpp

```diff
@@ -72,7 +74,12 @@ class TargetRewrite : public TargetRewriteBase<TargetRewrite> {
   void runOnOperation() override final {
     auto &context = getContext();
     mlir::OpBuilder rewriter(&context);
+
     auto mod = getModule();
+    if (!forcedTargetTriple.empty()) {
+      setTargetTriple(mod, forcedTargetTriple);
+    }
```

Comment:
nit: braces are not needed.

Reply:
They're not in LLVM, but I think they are in flang: https://github.com/flang-compiler/f18-llvm-project/blob/1fe8993ad81900eea4d0f5603dffd239e520c31d/flang/docs/C%2B%2Bstyle.md?plain=1#L117

Reply:
files in the `Optimizer` directory are actually following the MLIR guidelines (see note in header in this file).

Reply:
Riiight, I completely forgot about that.

Reply:
No worries. Hard to keep track of the different flavor sometimes :-)